packages/client/src/utils/index.ts

```diff
@@ -15,6 +15,20 @@ export const getSlugFromPathname = (pathname: string) => {
   return match?.params?.slug;
 };
 
+// Get variable from query string
+// TODO: add tests
+export const getQueryVariable = (variable: string) => {
+  var query = window.location.search.substring(1);
+  var vars = query.split("&");
+  for (var i = 0; i < vars.length; i++) {
+    var pair = vars[i].split("=");
+    if (decodeURIComponent(pair[0]) === variable) {
+      return decodeURIComponent(pair[1]);
+    }
+  }
+  return;
```

Comment:
If there's only one query variable there is no `&` but only a `?`, eg: `url/?var=value`. I think it's nice to add a fallback for this to make this function more general.